gnomeExtensions.gsconnect build error on master

**Describe the bug**
gnomeExtensions.gsconnect fails to build on master branch (tested ef399ee03542dcc45c9f3572eb75d4a36abd589b):

```
$ nix-build -A gnomeExtensions.gsconnect
...
Installing /build/source/data/icons/org.gnome.Shell.Extensions.GSConnect-symbolic.svg to /nix/store/wj3mq774hp694vr4nknzss1nm7q60jas-gnome-shell-gsconnect-31/share/icons/hicolor/scalable/apps
Traceback (most recent call last):
  File "/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/lib/python3.7/site-packages/mesonbuild/mesonmain.py", line 129, in run
    return options.run_func(options)
  File "/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/lib/python3.7/site-packages/mesonbuild/minstall.py", line 521, in run
    installer.do_install(datafilename)
  File "/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/lib/python3.7/site-packages/mesonbuild/minstall.py", line 346, in do_install
    self.install_data(d)
  File "/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/lib/python3.7/site-packages/mesonbuild/minstall.py", line 375, in install_data
    d.dirmaker.makedirs(outdir, exist_ok=True)
  File "/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/lib/python3.7/site-packages/mesonbuild/minstall.py", line 55, in makedirs
    os.makedirs(path, exist_ok=exist_ok)
  File "/nix/store/hmcha8bv2ziwabla76hd612vgxr20pnd-python3-3.7.6/lib/python3.7/os.py", line 211, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/nix/store/hmcha8bv2ziwabla76hd612vgxr20pnd-python3-3.7.6/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/etc/dbus-1'
FAILED: meson-install
/nix/store/vyh6mfdir6j2i0abpsdvm9ybcnlffbxy-meson-0.52.1/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
builder for '/nix/store/5fr6cmyny5zrh7w0sjyzv06h21bl8hvl-gnome-shell-gsconnect-31.drv' failed with exit code 1
error: build of '/nix/store/5fr6cmyny5zrh7w0sjyzv06h21bl8hvl-gnome-shell-gsconnect-31.drv' failed
```

**To Reproduce**
On master branch (e.g. ef399ee03542dcc45c9f3572eb75d4a36abd589b), run  `nix-build -A gnomeExtensions.gsconnect`.

**Expected behavior**
I expect packages to (continue to) build and not break when trying to upgrade my system.

**Additional context**
I ran git bisect and it pointed to d1dfbfd8 (2020-01-06). I'm quite sure that's not the commit that caused the current breakage, but that's where gsconnect stopped building initially.

**Metadata**
```
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.98, NixOS, 20.03.git.804fbc9 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.2`
 - channels(root): `"nixos-19.09.1840.f7d050ed4e3"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```
Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/desktops/gnome-3/extensions/gsconnect/default.nix
+++ b/pkgs/desktops/gnome-3/extensions/gsconnect/default.nix
@@ -49,6 +49,7 @@ stdenv.mkDerivation rec {
     "-Dopenssl_path=${openssl}/bin/openssl"
     "-Dsshadd_path=${openssh}/bin/ssh-add"
     "-Dsshkeygen_path=${openssh}/bin/ssh-keygen"
+    "-Dsession_bus_services_dir=${placeholder "out"}/share/dbus-1/services"
     "-Dpost_install=true"
   ];
 

